test/public-stats.test.js

```diff
@@ -627,6 +629,198 @@ describe('public-stats', () => {
+        assert.deepStrictEqual(created, [])
+        await updateDailyClientRetrievalStats(
+          pgClient,
+          committees,
+          findDealClients
+        )
+        const { rows: f0Stats } = await pgClient.query(
+          "SELECT day::TEXT,client_id,total,successful,successful_http FROM daily_client_retrieval_stats WHERE client_id = 'f0client'"
+        )
+        assert.strictEqual(f0Stats.length, 1)
+        assert.deepStrictEqual(f0Stats, [
+          { day: today, client_id: 'f0client', total: 2, successful: 2, successful_http: 2 }
+        ])
+        const { rows: f1Stats } = await pgClient.query(
+          "SELECT day::TEXT,client_id,total,successful,successful_http FROM daily_client_retrieval_stats WHERE client_id = 'f1client'"
+        )
+        // All measurments are successful and successful_http
+        assert.strictEqual(f1Stats.length, 1)
+        assert.deepStrictEqual(f1Stats, [
+          { day: today, client_id: 'f1client', total: 3, successful: 3, successful_http: 3 }
+        ])
+      })
+      it('aggregates overlapping per client stats', async () => {
+        /** @type {Measurement[]} */
+        const allMeasurements = [
+          // minerId f01test stores deals for both client f0 and f1
+          { ...VALID_MEASUREMENT, protocol: 'http', minerId: 'f01test' },
+          { ...VALID_MEASUREMENT, protocol: 'http', minerId: 'f01test' },
+
+          { ...VALID_MEASUREMENT, protocol: 'http', minerId: 'f1test' },
+          { ...VALID_MEASUREMENT, protocol: 'http', minerId: 'f1test' },
+          { ...VALID_MEASUREMENT, protocol: 'http', minerId: 'f1test' }
+        ]
+
+        // Separate the measurments into two groups, one for the client f0 and f1 and the other only for f1
+        const findDealClients = (minerId, _cid) => {
+          switch (minerId) {
+            case 'f01test':
+              return ['f0client', 'f1client']
+            case 'f1test':
+              return ['f1client']
+            default:
+              throw new Error('Unexpected minerId')
+          }
+        }
+        const committees = buildEvaluatedCommitteesFromMeasurements(allMeasurements)
+        await updateDailyClientRetrievalStats(
+          pgClient,
+          committees,
+          findDealClients
+        )
+        const { rows: f0Stats } = await pgClient.query(
+          "SELECT day::TEXT,client_id,total,successful,successful_http FROM daily_client_retrieval_stats WHERE client_id = 'f0client'"
+        )
+        assert.deepStrictEqual(f0Stats, [
+          { day: today, client_id: 'f0client', total: 2, successful: 2, successful_http: 2 }
+        ])
+        const { rows: f1Stats } = await pgClient.query(
+          "SELECT day::TEXT,client_id,total,successful,successful_http FROM daily_client_retrieval_stats WHERE client_id = 'f1client'"
+        )
+        assert.deepStrictEqual(f1Stats, [
+          { day: today, client_id: 'f1client', total: 5, successful: 5, successful_http: 5 }
+        ])
```

Comment:
```suggestion
        assert.deepStrictEqual(f1Stats, [
                  { day: today, client_id: 'f0client', total: 2, successful: 2, successful_http: 2 },
          { day: today, client_id: 'f1client', total: 5, successful: 5, successful_http: 5 }
        ])
```

Continuation of [previous comment](https://github.com/CheckerNetwork/spark-evaluate/pull/481/files#r1963014626)

Reply:
I see that I made an error in this code suggestion, `f1Stats` should actually be `rows`

Reply:
I addressed this in bc27e9a